### PR TITLE
update add nodes

### PR DESCRIPTION
**Pull request by Forest-L**
Signed-off-by: “Forest-L <lilin@yunify.com>

### CODE REVIEW

File: pkg/add/add.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ import (
 	"path/filepath"
 )
 
-func ScaleCluster(clusterCfgFile, k8sVersion, ksVersion string, logger *log.Logger, ksEnabled, verbose, skipCheck, skipPullImages bool) error {
+func AddNode(clusterCfgFile, k8sVersion, ksVersion string, logger *log.Logger, ksEnabled, verbose, skipCheck, skipPullImages bool) error {
```
Comment (rayzhou2017) [inline pkg/add/add.go:35]: AddNodes

File: cmd/add_node.go [hunk 2]
```diff
@@ -16,24 +16,24 @@ limitations under the License.
...
 	"github.com/kubesphere/kubekey/pkg/util"
 	"github.com/spf13/cobra"
 )
 
-// scaleCmd represents the scale command
-var scaleCmd = &cobra.Command{
-	Use:   "scale",
-	Short: "Scale a cluster according to the new nodes information from the specified configuration file",
+// addNodeCmd represents the nodes command
+var addNodeCmd = &cobra.Command{
```
Comment (rayzhou2017) [inline cmd/add_node.go:25]: addNodesCmd

File: cmd/add_node.go [hunk 3]
```diff
@@ -16,24 +16,24 @@ limitations under the License.
...
 )
 
-// scaleCmd represents the scale command
-var scaleCmd = &cobra.Command{
-	Use:   "scale",
-	Short: "Scale a cluster according to the new nodes information from the specified configuration file",
+// addNodesCmd represents the nodes command
+var addNodesCmd = &cobra.Command{
+	Use:   "nodes",
+	Short: "Add a cluster according to the new nodes information from the specified configuration file",
```
Comment (pixiake) [inline cmd/add_node.go:27]: add nodes to cluster

**Pull request merged**